qa/L0_model_config/autofill_noplatform/python/dynamic_batching_no_op/expected

```diff
@@ -0,0 +1 @@
+Configuration already has scheduler 'dynamic_batching' and will not auto-complete scheduler to dynamic_batching.
```

Comment:
This error does not make sense. Why are we throwing error here if the configuration is already providing the scheduler choice as 'dynamic_batching'?

Reply:
This is related to [this thread](https://github.com/triton-inference-server/python_backend/pull/165#discussion_r891619472) where the decision was made to throw an error in this case.  I will update this test in accordance with the linked thread.

Reply:
moved to success directory